src/compiler/scala/tools/nsc/backend/jvm/GenBCodeOps.scala

```diff
@@ -0,0 +1,21 @@
+/* NSC -- new Scala compiler
+ * Copyright 2005-2012 LAMP/EPFL
+ * @author  Martin Odersky
+ */
+
+
+package scala
+package tools.nsc
+package backend
+package jvm
+
+import scala.tools.asm
+
+object GenBCodeOps extends GenBCodeOps
```

Comment:
As in the other cases, I needed to compile the object but not the class, but kept a compatibility `object GenBCode extends GenBCodeOps`